src/default_metrics.go

```diff
@@ -0,0 +1,127 @@
+	"query.comRollbackPerSecond":                  {"Com_rollback", metric.RATE},
+	"query.comSelectPerSecond":                    {"Com_select", metric.RATE},
+	"query.comUpdatePerSecond":                    {"Com_update", metric.RATE},
+	"query.comUpdateMultiPerSecond":               {"Com_update_multi", metric.RATE},
+	"db.handlerRollbackPerSecond":                 {"Handler_rollback", metric.RATE},
+	"query.preparedStmtCountPerSecond":            {"Prepared_stmt_count", metric.RATE},
+	"query.queriesPerSecond":                      {"Queries", metric.RATE},
+	"query.questionsPerSecond":                    {"Questions", metric.RATE},
+	"query.slowQueriesPerSecond":                  {"Slow_queries", metric.RATE},
+	"db.innodb.bufferPoolPagesData":               {"Innodb_buffer_pool_pages_data", metric.GAUGE},
+	"db.innodb.bufferPoolPagesFree":               {"Innodb_buffer_pool_pages_free", metric.GAUGE},
+	"db.innodb.bufferPoolPagesTotal":              {"Innodb_buffer_pool_pages_total", metric.GAUGE},
+	"db.innodb.dataReadBytesPerSecond":            {"Innodb_data_read", metric.RATE},
+	"db.innodb.dataWrittenBytesPerSecond":         {"Innodb_data_written", metric.RATE},
+	"db.innodb.logWaitsPerSecond":                 {"Innodb_log_waits", metric.RATE},
+	"db.innodb.rowLockCurrentWaits":               {"Innodb_row_lock_current_waits", metric.GAUGE},
+	"db.innodb.rowLockTimeAvg":                    {"Innodb_row_lock_time_avg", metric.GAUGE},
+	"db.innodb.rowLockWaitsPerSecond":             {"Innodb_row_lock_waits", metric.RATE},
+	"db.openFiles":                                {"Open_files", metric.GAUGE},
+	"db.openTables":                               {"Open_tables", metric.GAUGE},
+	"db.openedTablesPerSecond":                    {"Opened_tables", metric.RATE},
+	"db.tablesLocksWaitedPerSecond":               {"Table_locks_waited", metric.RATE},
+	"software.edition":                            {"version_comment", metric.ATTRIBUTE},
+	"software.version":                            {"version", metric.ATTRIBUTE},
+	"cluster.nodeType":                            {"node_type", metric.ATTRIBUTE},
+}
+
+var defaultMetrics560 = map[string][]interface{}{
+	"db.qCacheFreeMemoryBytes":    {"Qcache_free_memory", metric.GAUGE},
+	"db.qCacheNotCachedPerSecond": {"Qcache_not_cached", metric.RATE},
+	"db.qCacheUtilization":        {qCacheUtilization, metric.GAUGE},
+	"db.qCacheHitRatio":           {qCacheHitRatio, metric.GAUGE},
+	// If a cluster instance is not a slave, then the metric cluster.slaveRunning will be removed.
+	"cluster.slaveRunning": {slaveRunningAsNumber, metric.GAUGE},
+}
+
+var defaultMetrics800 = map[string][]interface{}{
+	// If a cluster instance is not a slave, then the metric cluster.slaveRunning will be removed.
+	"cluster.slaveRunning": {slaveRunningAsNumber, metric.GAUGE},
+}
+
+func slaveRunningAsNumber(metrics map[string]interface{}, version *semver.Version) (int, bool) {
```

Comment:
I think this requires unit tests.

Reply:
This is already existing func just moved to diff file